provides.py

```diff
@@ -37,12 +37,12 @@ def joined_or_changed(self):
 
     @hook('{provides:kube-control}-relation-{broken,departed}')
```

Comment:
The `-broken` hook completely fails with the current conversation model for interface layers because there is no `REMOTE_UNIT` and that's not handled well.  It also doesn't have a real use in the reactive pattern because you should be removing the states on a per-unit basis anyway and `-broken` just becomes the state being removed from the last conversation and thus starting to match `@when_not` again.

TL;DR: interface layers should only ever use `-departed` and using `-broken` can cause some strange issues.